rero_ils/modules/loans/listener.py

```diff
@@ -17,6 +17,7 @@
 
 """Signals connector for Loan."""
 
+from flask.globals import current_app
```

Comment:
Why not `from flask import current_app` ?

Reply:
Changed it, this was old code.